projects/ngx-launcher/src/lib/create-app/project-progress-createapp-nextstep/project-progress-createapp-nextstep.component.ts

```diff
@@ -15,13 +20,18 @@ import { Broadcaster } from 'ngx-base';
 export class ProjectProgressCreateappNextstepComponent implements OnChanges, OnDestroy {
   @Input() statusLink: string;
   errorMessage: string;
+  codeBaseCreated = false;
+  codebaseId: string;
   private _progress: Progress[];
   private socket: WebSocket;
 
   constructor(@Host() public launcherComponent: LauncherComponent,
     private projectProgressService: ProjectProgressService,
     private projectSummaryService: ProjectSummaryService,
-    private broadcaster: Broadcaster) {
+    private broadcaster: Broadcaster,
+    private workSpaceService: WorkSpacesService,
```

Comment:
Perhaps it would be better to annotate these new services with `@Optional`, otherwise 'launch' will need implementations for these services as well